include/openenclave/bits/sgx/seal.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/**
+ * @file sgxproperties.h
+ *
+ * This file defines the SGX properties for an enclave.
+ *
+ * The enclave properties should only be defined once for all code compiled
+ * into an enclave binary using the OE_SET_ENCLAVE_SGX macro.
+ * These properties can be overwritten at sign time by the oesign tool.
+ */
+
+#ifndef _OE_BITS_SGX_SEAL_H
+#define _OE_BITS_SGX_SEAL_H
+
+#include <openenclave/bits/sgx/sgxtypes.h>
+
+#define OE_SEAL_SGX_SPECIFIC    1ULL
+
+typedef struct _oe_sealed_blob_header
+{
+    sgx_key_request_t   key_info;
+    uint32_t            aad_offset;
```

Comment:
please define "aad", at least in a /* comment */, to make this more readable.
```suggestion
    uint32_t            aad_offset; /* byte offset of the Additional Authenticated Data in the sealed blob */
```